vaadin-form-layout-flow-parent/vaadin-form-layout-flow/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java

```diff
@@ -279,6 +279,86 @@ public void remove(Component... components) {
         }
     }
 
+    /**
+     * Server-side component for the {@code <vaadin-form-row>} element. Used to
+     * arrange fields into rows inside a {@link FormLayout} when
+     * {@link FormLayout#setAutoResponsive(boolean) auto-responsive mode} is
+     * enabled.
+     * <p>
+     * Each FormRow always starts on a new row. Fields that exceed the available
+     * columns wrap to a new row, which then remains reserved exclusively for
+     * the fields of that FormRow.
+     * <p>
+     * Example of creating a FormRow with two fields and a single field that
+     * spans two columns:
+     *
+     * <pre>
+     * FormLayout formLayout = new FormLayout();
+     * formLayout.setAutoResponsive(true);
+     * formLayout.addFormRow(new TextField("First name"),
+     *         new TextField("Last name"));
+     *
+     * TextArea addressField = new TextArea("Address");
+     * formLayout.setColspan(addressField, 2);
+     * formLayout.addFormRow(addressField);
+     * </pre>
+     *
+     * @author Vaadin Ltd
+     */
+    @Tag("vaadin-form-row")
+    @NpmPackage(value = "@vaadin/polymer-legacy-adapter", version = "24.7.0-beta1")
+    @JsModule("@vaadin/polymer-legacy-adapter/style-modules.js")
+    @NpmPackage(value = "@vaadin/form-layout", version = "24.7.0-beta1")
+    @JsModule("@vaadin/form-layout/src/vaadin-form-row.js")
+    public static class FormRow extends Component
+            implements HasComponents, HasStyle {
```

Comment:
nit: the Component already implements HasStyle since https://github.com/vaadin/flow/pull/15742.

Reply:
Removed!